src/state/__tests__/reducer.ts

```diff
@@ -261,4 +262,28 @@ describe('reducer', () => {
     const newState = reducer(iniState, action);
     expect(newState).toBe(iniState);
   });
+  it('reset should delete all entities', () => {
```

Comment:
nit: check here is more than just entities so maybe call it 'reset should delete all entries'